guides/doc-Provisioning_Guide/topics/proc_building-a-discovery-image.adoc

```diff
@@ -0,0 +1,119 @@
+[[building-a-discovery-image]]
+= Building a Discovery Image
+
+The Discovery image is a minimal operating system based on {RHEL} that is PXE-booted on hosts to acquire initial hardware information and to check in to the Satellite Server. Discovered hosts keep running the Satellite Discovery image until they are rebooted into Anaconda, which then initiates the provisioning process. The `foreman-discovery-image` package contains this image. You must install the package on the {SmartProxy} that provides TFTP services.
+
+Use this procedure to build a {Project} discovery image or rebuild an image if you change configuration files.
+
+Do not use this procedure on your production {Project} or {SmartProxy}.
+
+.Prerequisites
+
+Install the `livecd-tools` package:
+ifeval::["{build}" == "foreman"]
+----
+# yum install livecd-tools
+----
+endif::[]
+ifeval::["{build}" == "satellite"]
+[options="nowrap" subs="+quotes,attributes"]
+----
+# {foreman-maintain} packages install livecd-tools
+----
+endif::[]
+
+Because Anaconda installer cannot publish through HTTPS, you must enable publishing through HTTP for Kickstart repositories:
+
+. In the {Project} web UI, navigate to *Content* > *Products* and in the *Products* window, click the *Repositories* tab.
+. Select a Kickstart repository and for the *Publish via HTTP*, option, click the *Edit* icon, select the check box, and click *Save*.
+. Repeat the previous steps for the {Project} repository.
+
+Note that publishing via HTTP does not apply to any Red{nbsp}Hat repositories.
```

Comment:
This whole block (lines 25-31) only applies when Satellite or Katello is installed. So I'd probably drop a sentence before with if-foreman block: When building FDI from content that is not managed by {Project}, skip the following paragraph.